src/GrumPHP/Configuration/GrumPHP.php

```diff
@@ -119,4 +120,12 @@ public function isBlockingTask($taskName)
         $taskMetadata = $this->getTaskMetadata($taskName);
         return $taskMetadata['blocking'];
     }
+
+    /**
+     * @return Filesystem
+     */
+    public function getFilesystem()
```

Comment:
Can you add a spec for this?
Maybe it's better to inject the Filesystem class directly in the task?